doc/nrf/protocols/matter/end_product/configuring_cd.rst

```diff
@@ -0,0 +1,46 @@
+.. _ug_matter_device_configuring_cd:
+
+Configuring Certification Declaration in firmware
+#################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Certification Declaration (CD) is :ref:`one of the documents <ug_matter_device_certification_results>` obtained as a result of a successful Matter certification.
+It is |matter_cd_definition|.
+It is used for this purpose in the :ref:`ug_matter_device_attestation` procedure during commissioning, where it is provided to a device as :ref:`Device Attestation data <ug_matter_device_attestation_device_data>`.
+
+The CD is an array of bytes and is part of the firmware that is allowed to be modified and regenerated after the certification.
+You do not have to apply again for certification if you only modify the CD.
+
+Why modify the CD
+*****************
+
+
+
+Modifying the CD
+****************
+
+In the |NCS| implementation of Matter, you can configure the CD by defining the :c:macro:`CHIP_DEVICE_CONFIG_CERTIFICATION_DECLARATION` define in the :file:`chip_project_config.h` file.
+You can locate the array of bytes related to the CD by running the search with the following condition:
+
+.. code-block:: console
+
+   cat CD.der | xxd -I
+
+Storing the CD in Zephyr
+========================
+
+Alternatively, you can opt for storing the CD in the Zephyr's Settings subsystem.
+In such case, you must set the :kconfig:option:`CONFIG_CHIP_CERTIFICATION_DECLARATION_STORAGE` KConfig option instead of the define.
+
+Regenerating the CD
+*******************
+
+To regenerate the CD, contact `Connectivity Standards Alliance`_ and provide them with the following information:
```

Comment:
@Damian-Nordic , @ArekBalysNordic , why would I want to do this?

Reply:
Explained above why it is needed. When the customer has the new software for a device they must contact CSA and provide all of the following information.